api/serializers.py

```diff
@@ -236,16 +254,19 @@ def validate(self, attrs):
         # shifts in that month for that contract
         if exported_shifts:
             raise serializers.ValidationError(
-                "Für diesen Monat wurde bereits ein Stundenzettel exportiert.\n"
-                "Es ist nicht möglich Schichten in diesem Monat zu ändern oder neue zu erstellen. "
+                _(
+                    "A worktime-sheet for this month has already been exported.\n"
+                    "It is not possible to add or modify shifts."
+                )
             )
 
         return attrs
 
     def validate_contract(self, contract):
         if not (contract.user == self.context["request"].user):
             raise serializers.ValidationError(
-                "Das Vertragsobjekt muss dem User gehören der die Schicht erstellt."
+                _("The contract object must be owned by the user creating the shift."
```

Comment:
```suggestion
                _("The contract must be owned by the user creating the shift."
```

Das sind ja Fehler die im Zweifel den Usern angezeigt werden. Da sollte kein "Objekt" vorkommen.